src/framework/uicomponents/qml/MuseScore/UiComponents/StyledMenuItem.qml

```diff
@@ -16,6 +16,7 @@ ListItemBlank {
 
     property var modelData
     property bool reserveSpaceForInvisibleItems: true
+    property bool unifyCheckMarksAndIcons: true
```

Comment:
I don't quite understand what this flag does. Can you explain me, please?

Reply:
When this is true, the checkmark of a checked action appears _instead of_ the icon.
When it is false, the checkmark appears _on the left of_ the icon. 
| unifyCheckMarksAndIcons = true | | unifyCheckMarksAndIcons = false |
|-|-|-|
|<img width="300" alt="Schermafbeelding 2021-04-11 om 16 03 36" src="https://user-images.githubusercontent.com/48658420/114307345-7c604300-9adf-11eb-87bf-472ba74a7dc5.png">|vs|<img width="300" alt="Schermafbeelding 2021-04-11 om 16 00 31" src="https://user-images.githubusercontent.com/48658420/114307238-0c51bd00-9adf-11eb-9e69-fb1e60ada66d.png">|

Reply:
Oh, I see. But do we really need to display the checkmark instead of the icon somewhere?

Reply:
It seems that we should always display both the checkmark and the icon (correct me if I'm wrong)

Reply:
Maybe that's a question for @Tantacrul. I did this to keep the existing behaviour in other places, but maybe the existing behaviour was not correct.

Reply:
I think that better to remove this flag and always display both the checkmark and the icon. We can change this behaviour later if need be

Reply:
Oops, messages crossed :) 
I will do it like this:
- if any item in a menu has a checkmark, all items have a "column" for checkmarks
- if any item has an icon, all items will have a "column" for icons
- but if all items that are checkable don't have an icon, the checkmarks and icons are in the same "column" because that saves space.

Reply:
> Oops, messages crossed :)
> I will do it like this:
> 
> * if any item in a menu has a checkmark, all items have a "column" for checkmarks
> * if any item has an icon, all items will have a "column" for icons
> * but if all items that are checkable don't have an icon, the checkmarks and icons are in the same "column" because that saves space.

That makes sense to me. This kind of flexibility in context menus can be seen on a few other apps and OS's too.